testing/src/test/resources/bootstrap.properties

```diff
@@ -0,0 +1,19 @@
+sql.url=jdbc:mysql://localhost:3306/
+sql.user=root
+sql.password=password
+tenant.id=mtenant
+config.source=UI
+sql.driver=com.mysql.jdbc.Driver
+
+########################################################################
+#AcitveMQ properties
+########################################################################
+jms.queue.for.events=QueueForEvents
+jms.queue.for.scheduler=QueueForScheduler
+jms.broker.url=tcp://localhost:61616
+jms.maximumRedeliveries=0
+jms.redeliveryDelayInMillis=2000
+jms.concurrentConsumers=1
+jms.maxConcurrentConsumers=10
+jms.session.cache.size=10
+jms.cache.producers=false
```

Comment:
Koshal suggested that this was the cause for the sporadic total failure due to the framework not starting.